policyengine_us/parameters/gov/states/ct/tax/income/main/add_back/amount.yaml

```
@@ -0,0 +1,22 @@
+description: Connecticut phases its 3% income tax rate out by this amount, based on filing status.
```

And elsewhere - avoid including parameter values in descriptions and labels.
```suggestion
description: Connecticut phases its bottom income tax rate out by this amount, based on filing status.
```